pkgs/games/minetest/default.nix

```diff
@@ -90,8 +91,10 @@ stdenv.mkDerivation (finalAttrs: {
     ncurses
     gmp
     libspatialindex
-  ] ++ lib.optional (lib.meta.availableOn stdenv.hostPlatform luajit) luajit
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+  ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform luajit) [
+    luajit
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
This change is not needed. The condition is on the availability of luajit, it's not like we are going to add anything else to the list other than luajit itself

Reply:
Reverted.